backend/src/Controllers/elections.controllers.ts

```diff
@@ -2,21 +2,23 @@ import { Election } from '../../../domain_model/Election';
 import { Ballot } from '../../../domain_model/Ballot';
 import { Score } from '../../../domain_model/Score';
 import ServiceLocator from '../ServiceLocator';
+import Logger from '../Services/Logging/Logger';
+import { responseErr } from '../Util';
 const ElectionsDB = require('../Models/Elections')
 const StarResults = require('../Tabulators/StarResults.js');
 
 var ElectionsModel = new ElectionsDB(ServiceLocator.postgres());
+var className="Elections.Controllers";
```

Comment:
Instead of a className variable, could we retrieve the filename dynamically and then bake that logic into our logging library?

https://stackoverflow.com/questions/14201475/node-js-getting-current-filename

Reply:
AFAIK, using __filename in the logging library would just return the LoggerImpl for everything, which is not helpful.

I've done what you're suggesting in other languages, so I assume it can be done.  A first glance at TS made it seem complicated so we can add that as a "nice to have"?